src/Options/Definitions.js

```diff
@@ -28,6 +28,13 @@ module.exports.SchemaOptions = {
     action: parsers.booleanParser,
     default: false,
   },
+  dropUnknownIndexes: {
+    env: 'PARSE_SERVER_SCHEMA_DROP_UNKNOWN_INDEXES',
+    help:
+      'Drops indexes that are not defined in the schema and are present in the database. Set this false if you are adding indexes manually so that it wont be dropped when you run schema migration',
+    action: parsers.booleanParser,
+    default: true,
```

Comment:
issue: if you are in default: true, it means the var is inversed if we keep the current behavior of defined schema.

Here the var should be changed to `keepUnknowIndexes`, and default to false